python/ray/_private/node.py

```diff
@@ -1014,6 +1017,7 @@ def start_raylet(
             self._ray_params.node_manager_port,
             self._raylet_socket_name,
             self._plasma_store_socket_name,
+            self.cluster_id.decode(),
```

Comment:
Why do we need to `decode()`? Is `self.cluster_id` the hex string?

Reply:
Yes :) At some point it was getting serialized with an extra 'b' in front

Reply:
let's make sure `get_cluster_id()` returns the decoded hex string?